templates/scripts/post-apply.sh

```diff
@@ -1,4 +1,20 @@
 #!/bin/sh
-set -e
+
+KUBE_CONTEXT=${PROJECT}-${ENVIRONMENT}-${AWS_DEFAULT_REGION}
```

Comment:
I notice we're no longer using the region from `parameters` but using `AWS_DEFAULT_REGION` instead
would this cause issues where the user's AWS profile has a different default region than the one picked?

Reply:
The makefile is setting `AWS_DEFAULT_REGION` to the content of the `region` parameter.